awx/ui_next/src/components/FormLayout/FormLayout.jsx

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components';
+
+export const FormColumnLayout = styled.div`
+  width: 100%;
+  grid-column: 1 / 4;
```

Comment:
`grid-column: 1 / -1`

Reply:
this one has to be 1 / 4 as a child of patternfly's Form component...1 / -1 does not work for some reason.

Reply:
nevermind, looks like this works now.  must be due to another change that happened since i tested this, great!